src/iden3comm/types/protocol/contract-request.ts

```diff
@@ -45,10 +52,21 @@ export type ContractInvokeTransactionData = {
   network?: string;
 };
 
+export type AuthProofEthIdentity = {
+  userDid: DID;
```

Comment:
Any convention about when to use `userDid` or `userDID` ? `userDid` when it's a property of object type and `userDID` for function params and variables?